frontend-v2/src/stories/Model.stories.tsx

```diff
@@ -363,7 +363,9 @@ export const HillCoefficient: Story = {
     expect(hillCoefficientCheckbox).toBeChecked();
 
     await selectMenuOption(pdModelList, "tumour_growth_linear", userEvent);
-    await waitForElementToBeRemoved(hillCoefficientCheckbox);
+    await waitForElementToBeRemoved(hillCoefficientCheckbox, {
+      timeout: 2000, // the default timeout isn't long enough in CI.
+    });
```

Comment:
This should be on the main branch too. CI intermittently fails with the default timeout.